ignite/src/main/java/org/hibernate/ogm/datastore/ignite/IgniteProperties.java

```diff
@@ -29,6 +32,36 @@
 	 */
 	public static final String IGNITE_INSTANCE_NAME = "hibernate.ogm.ignite.instance_name";
 
+	public static final String IGNITE_ALLOWS_TRANSACTION_EMULATION = "hibernate.ogm.ignite.allows_transaction_emulation";
```

Comment:
By the way,  this property is ignored by OGM if you are using a custom `TransactionCoordinatorBuilder`. Please remove it